src/main/cluster.ts

```diff
@@ -130,7 +130,7 @@ export class Cluster implements ClusterModel {
     if (!this.eventDisposers.length) {
       this.bindEvents();
     }
-    if (this.disconnected || !this.accessible) {
+    if (this.disconnected) {
```

Comment:
Why `!this.accessible` is removed? Is it causing some issues?

Reply:
Yes, it was causing double connect to cluster on activation, but now seems to works fine. Reverted.